Rails/Forms/odin_project_forms/app/views/users/new.html.erb

```diff
@@ -0,0 +1,12 @@
+<%= form_with(model: @user, local: true) do |f| %>
+  <%= f.label :username, "Enter a username" %>
+  <%= f.text_field :username, placeholder: "Sample Name" %>
+  <br><br>
+  <%= f.label :email %>
+  <%= f.text_field :email, value: "test@example.com" %>
+  <br><br>  
+  <%= f.label :password %>
+  <%= f.text_field :password %>
```

Comment:
Password field is currently a `text_field` so the password isn't being obscured on user entry:

 ![](https://screenshot.click/hi23-43-ll9g5nu-o0addgpck.png)

Reply:
Nice catch, @richardtshop! I've updated this to be `password_field`.